### ISSUE TITLE
Improve many_sprites example

### ISSUE THREAD

**Issue by alice-i-cecile** [bevyengine/bevy#2075]
## How can Bevy's documentation be improved?

[Link](https://github.com/bevyengine/bevy/blob/main/examples/2d/many_sprites.rs).

Things that could be better:
1. It's unclear why we need both a `Transform` and `Position` component. Usually, `Transform` itself contains this information.
2. `move_camera` should use a query filter along with `query.single` to only move, well, the camera.
3. Use `spawn_batch` for spawning many sprites.

**Comment by willolisp** [comment]
Well, this one looks simple enough that I could give it a shot, if there's nobody else working on it.

**Comment by alice-i-cecile** [comment]
@willolisp that sounds great :) I don't think anyone else is tackling this right now so feel free!

**Comment by willolisp** [comment]
@alice-i-cecile hmmm actually, looks like #2078 already fixes this and is just waiting for a more thorough review? Should I just close my PR?

**Comment by alice-i-cecile** [comment]
@willolisp right, thanks for the reminder. Sorry for missing this; duplicating work always feels bad 😅

I slightly prefer the changes in #2785; I think it's a bit more direct in a few places (like the timer system) and doesn't rely on hard-to-explain constants for positioning. When merging, we should give credit to both you and @neosam in some fashion.

**Comment by willolisp** [comment]
@alice-i-cecile sure, no worries! Thanks for the reply and for the review.
